hokusai/staging.yml

```diff
@@ -52,7 +60,7 @@ spec:
               containerPort: 8080
           resources:
             requests:
-              cpu: 100m
+              cpu: 50m
```

Comment:
[near zero](https://app.datadoghq.com/dashboard/2n5-6tr-ypp/kubernetes-cpumemory-usage-by-deployment?from_ts=1616080309689&live=true&tile_size=s&to_ts=1616685109689&tpl_var_deployment=convection-web&tpl_var_env=staging&tpl_var_hpa=convection-web) usage in staging.